src/vm/exceptionhandling.cpp

```diff
@@ -448,14 +448,7 @@ void ExceptionTracker::UpdateNonvolatileRegisters(CONTEXT *pContextRecord, REGDI
     UPDATEREG(X26);
     UPDATEREG(X27);
     UPDATEREG(X28);
-    // Obtain value of Fp from CurrentContext instead of from CurrentContextPointers
-    // It should not matter. CurrentContextPointers does not have value of FP as this will 
-    // require changes in MachState to also store pointer of FP which it does not do currently.
-    pContextRecord->Fp = pRegDisplay->pCurrentContext->Fp;
-    if (pAbortContext)
-    {
-        pAbortContext->Fp = pContextRecord->Fp;
-    }
+    UPDATEREG(Fp);
```

Comment:
I guess I am actually mistaken. I have read the comment that was there before as "the unwinder doesn't set the context pointer for Fp", which would mean that it is not being set in the unwinder inside of windows that we use. But reading it again, it seems that it actually means that it just requires changes to the MachState that you have made, so it should probably be ok.
@rahku, @gkhanna79 - am I right?